lib/service/storage/elasticsearch.ts

```diff
@@ -2296,16 +2516,23 @@ class ElasticSearch extends Service {
    * @returns {Promise.<{ items, errors }>
    */
   async mCreateOrReplace(
-    index,
-    collection,
-    documents,
+    index: string,
+    collection: string,
+    documents: JSONObject[],
     {
       refresh,
       timeout,
       userId = null,
       injectKuzzleMeta = true,
       limits = true,
       source = true,
+    }: {
+      refresh?: string;
+      timeout?: number;
+      userId?: string;
+      injectKuzzleMeta?: boolean;
+      limits?: boolean;
+      source?: boolean;
```

Comment:
Maybe define an interface with the requests options in `lib/types/storage/ElasticSearch.ts` and import it to use in methods parameters, it's more clear and lighter to read.

Reply:
you are right